src/SgidClient.ts

```diff
@@ -209,6 +213,28 @@ export class SgidClient {
     return { sub, data: {} }
   }
 
+  /**
+   * Generates dynamic derived data based on rules defined on sgID Developer Portal.
+   * @param rulesParams The parameters for dynamic derived data calculation.
+   * @param rulesParams.clientId The client ID of the Relying Party.
+   * @param rulesParams.accessToken The access token returned from the callback function.
+   * @param rulesParams.ruleNames The space-separated string containing rule names.
+   * @param rulesParams.userInfoData The end-user's verified data.
+   * @returns {RulesReturn} A list of rule names, alongside with their corresponding inputs and outputs.
+   */
+  async rules(rulesParams: RulesParams): Promise<RulesReturn> {
+    try {
+      const response = await fetch(SGID_RULES_ENGINE_URL, {
+        method: 'POST',
+        body: JSON.stringify(rulesParams),
+      })
```

Comment:
```suggestion
      const response = await fetch(SGID_RULES_ENGINE_URL, {
        method: 'POST',
        body: JSON.stringify(rulesParams),
        headers: {
           { Authentication: `Bearer ${accessToken}`}
        }
      })
```

Reply:
auth token (in this case accessToken, should be a bearer token instead of sending through body.

Reply:
Resolved in [6e9e0ca](https://github.com/opengovsg/sgid-client/pull/65/commits/6e9e0ca1445f1e14c59c7364dc0f9e9a382e8623).